spec/rubocop/cop/layout/empty_lines_around_method_body_spec.rb

```diff
@@ -19,12 +19,10 @@ def some_method
   # two cops registering offense for the line with only spaces would cause
   # havoc in auto-correction.
   it 'accepts method body starting with a line with spaces' do
-    expect_no_offenses([
-                         'def some_method',
-                         '  ',
-                         '  do_something',
-                         'end'
-                       ])
+    expect_no_offenses(['def some_method',
```

Comment:
Seems you forgot this.

Reply:
Ah, now I see. You just converted it to string. I expected you'd make everything a heredoc. :-)